allennlp/service/servable/models/bidaf.py

```diff
@@ -0,0 +1,50 @@
+from allennlp.common import Params, constants
+from allennlp.data import Vocabulary
+from allennlp.data.dataset_readers.squad import SquadReader
+from allennlp.data.fields import TextField
+from allennlp.data.tokenizers import WordTokenizer
+from allennlp.data.token_indexers import SingleIdTokenIndexer, TokenCharactersIndexer
+from allennlp.models import BidirectionalAttentionFlow
+from allennlp.service.servable import Servable, JSONDict
+
+class BidafServable(Servable):
+    def __init__(self):
+        constants.GLOVE_PATH = 'tests/fixtures/glove.6B.100d.sample.txt.gz'
+
+        reader_params = Params({
+                'token_indexers': {
+                        'tokens': {
+                                'type': 'single_id'
+                                },
+                        'token_characters': {
+                                'type': 'characters'
+                                }
+                        }
+                })
+        dataset = SquadReader.from_params(reader_params).read('tests/fixtures/squad_example.json')
+        vocab = Vocabulary.from_dataset(dataset)
+        self.vocab = vocab
+        print("vocab size", vocab.get_vocab_size())
+        dataset.index_instances(vocab)
+        self.dataset = dataset
+        self.token_indexers = {'tokens': SingleIdTokenIndexer(),
```

Comment:
Because you're constructing this here, it's probably easier to just pass these into the `SquadReader` constructor, instead of duplicating the token indexer specification.  Also, we'll want the `tokens` indexer to lowercase the word - glove uses a lowercase vocabulary.